codes/ruby/chapter_sorting/merge_sort.rb

```diff
@@ -0,0 +1,61 @@
+=begin
+File: merge_sort.rb
+Created Time: 2024-04-10
+Author: junminhong (junminhong1110@gmail.com)
+=end
+
+### 合并左子数组和右子数组 ###
+def merge(nums, left, mid, right)
+  # 左子数组区间为 [left, mid], 右子数组区间为 [mid+1, right]
+  # 创建一个临时数组 tmp，用于存放合并后的结果
+  tmp = Array.new(right - left + 1)
```

Comment:
It is better to preserve consistency with other codes. As in other codes, the `tmp` list is initialized as `[0, 0, 0, 0, 0]`

```suggestion
  tmp = Array.new(right - left + 1, 0)
```

Reply:
thanks

Reply:
[7cf1610](https://github.com/krahets/hello-algo/pull/1260/commits/7cf1610445510cba0d3282c5e73d5dffdea08253)